php/modules/Session.php

```diff
@@ -99,11 +112,18 @@ public function login($username, $password) {
 		$username = crypt($username, $this->settings['username']);
 		$password = crypt($password, $this->settings['password']);
 
-		# Check login with crypted hash
 		if ($this->settings['username']===$username&&
 			$this->settings['password']===$password) {
 				$_SESSION['login']		= true;
 				$_SESSION['identifier']	= $this->settings['identifier'];
+
+				$expire = time() + 60 * $this->settings['sessionLength'];
```

Comment:
sessionLength probably needs to be added to php/database/settings_content.sql. And a database update script created in the same directory that adds the default sessionLength to the settings table.